activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -36,6 +36,15 @@ def silence
         @silenced = old_silenced
       end
 
+
+      # Outputs a deprecation warning with 'will be removed' message to
```

Comment:
Sorry, I could not answer because I had a crazy week on my job and university. :sweat:
So, @kaspth, I absolutely agree with you and @matthewd and will add this now.